Resources/Prototypes/Entities/Items/wallstorage.yml

```diff
@@ -0,0 +1,25 @@
+- type: entity
+  name: fireaxe locker
+  id: WallFireaxeStorage
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Objects/wallstorage.rsi
+    drawdepth: Items
+  - type: Icon
+    sprite: Objects/wallstorage.rsi
+    state: empty
+  - type: Appearance
+    visuals:
+    - type: WallStorageVisualizer2D
```

Comment:
There's no such type, since you left the class called (and implemented as) 'PowerChargerVisualizer2D`